### PR TITLE
Add two new JS questions

### PR DESCRIPTION
**Pull request by Coakeow**
No idea about the answers. Will test later.

### CODE REVIEW

File: javascript/javascript-quiz.md [hunk 1]
```diff
@@ -1871,3 +1871,34 @@ console.log(cat.name);
...
+var thing;
+let func = (str = 'no arg') => {console.log(str);}
+func(thing);
+func(null);
+```
+
+- [ ] null no arg
+- [ ] no arg no arg
+- [ ] null null
+- [ ] no arg null
```
Comment (LazyKeru) [inline javascript/javascript-quiz.md:1887]: The answer `no arg null` Just execute the code in [Javascript Playground](https://playcode.io/)

File: javascript/javascript-quiz.md [hunk 2]
```diff
@@ -1871,3 +1871,34 @@ console.log(cat.name);
...
+};
+const a = 1;
+const test = myFunc();
+test();
+```
+
+- [ ] a is 1
+- [ ] a is undefined
+- [ ] It won't print anything.
+- [ ] a is 2
```
Comment (LazyKeru) [inline javascript/javascript-quiz.md:1904]: The answer `a is 2`. Just execute the code in [Javascript Playground](https://playcode.io/)

**Review by LazyKeru — CHANGES_REQUESTED** [review]

File: javascript/javascript-quiz.md (hunk 1 above)
Comment (Coakeow) [reply javascript/javascript-quiz.md]: Added

**Review by Ebazhanov — APPROVED** [review]
Thank you @Coakeow for the new questions with answers! Looks great! 💪

**Comment by Ebazhanov** [comment]
@all-contributors please add @Coakeow for design, code

**Pull request merged**